resqpy/grid/write_functions.py

```diff
@@ -0,0 +1,249 @@
+"""Submodule containing the functions relating to writing grid information."""
+
```

Comment:
I would prefer to split this module in two:
The write_hdf5 related functions are fundamental to RESQML
The write_nexus_corp() function is exporting to a different format, and is not so fundamental